spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+Rspec.describe Comment, type: :model do
+  test_user = User.create(name: 'yankee', bio: 'Teacher from Abuja')
+  test_post = Post.create(author: test_user, title: 'testing', text: 'thanks for testing this')
+  subject { Comment.new(author: test_user, post: test_post, text: 'hey thanks for having me ') }
+
+  it 'updates comment counter post' do
+    expect(subject.post.comments_counter).to be 1
+  end
```

Comment:
- [ ] You still need to provide the correct validations. Kindly change to something like this:

  ```
  test_user = User.create(name: 'yankee', bio: 'Teacher from Abuja', posts_counter: 0)
  test_post = Post.create(author: test_user, title: 'testing', text: 'thanks for testing this', comments_counter: 0, likes_counter: 0)
  subject { Comment.create(author_id: test_user.id, post_id: test_post.id, text: 'hey thanks for having me ') }

  it 'updates comment counter post' do
    expect(subject.post.comments_counter).to eq 1
  end
  ```